static/js/mediastream/peercall.js

```diff
@@ -142,6 +159,10 @@ define(['jquery', 'underscore', 'mediastream/utils', 'mediastream/peerconnection
 
 		this.setRemoteSdp(sessionDescription);
 
+		// Prevent creation of Offer messages to renegotiate streams while the
+		// remote Offer is being processed.
+		peerconnection.readyForRenegotiation = false;
```

Comment:
This should probably be a function, rather than directly setting an external member.
 Add a line note